tools/test/travis-ci/doxy-spellchecker/spell.sh

```diff
@@ -114,7 +114,9 @@ while read file; do
         echo "_________________________________"
     fi
 
-done < <(find "${1}" -type d -iname "*target*" -prune -o -name '*.h' -print)
+# ${1}: directory to check
+# ${2}: file containing a list of paths (regex) to exclude
+done < <(find "${1}" -type d -iname "*target*" -prune -o -name '*.h' -print | grep -v -f "${2}")
```

Comment:
Ideally the second argument should be optional, but I'm not sure how to do it after a pipe. Unless we rewrite this part in a different way.